src/app/modules/project/add/source/source.component.scss

```
@@ -41,21 +40,49 @@ p {
 }
 
 // sources
+.source-box {
+  position: relative;
+  padding: 0px;
+
+  &:before {
+    content: "";
+    display: block;
+    padding-top: 100%;
+  }
+}
+
 .source {
-  margin-top: 20px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  color: #fff;
+  line-height:100%;
+  height:100%;
```

Spaces between property name and value are preferred for clarity.
```suggestion
  line-height: 100%;
  height: 100%;
```